### DISCUSSION TITLE
adding picture on a note from galery

### DISCUSSION THREAD

**Discussion by Nono-m0le** [streetcomplete/StreetComplete#5106] [Feedback]
On the android app, when adding a note, I can only add a picture from the camera, but not from an already taken picture.
It would be nice to be able to do both.

**Comment by FloEdelmann** [comment]
See <https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Can_I_choose_photos_from_the_gallery_for_uploading_to_notes,_instead_of_using_camera?>
